renovate.json

```diff
@@ -1,5 +1,32 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": ["config:base"],
-  "enabledManagers": ["docker-compose","maven-wrapper","gradle-wrapper"]
+  "extends": [
+    "config:base",
+    ":separateMultipleMajorReleases",
+    ":automergeDigest",
+    "docker:pinDigests",
+    "docker:enableMajor"
+  ],
+  "enabledManagers": [
+    "docker-compose",
+    "maven-wrapper",
+    "gradle-wrapper"
+  ],
+  "packageRules": [
+    {
+      "matchDatasources": ["docker"],
+      "matchUpdateTypes": ["digest"],
+      "schedule": ["every weekend on the 1st through 7th day of the month"]
```

Comment:
why this schedule? it can run daily right?

Reply:
Yes, we agreed to follow PodiumD and that is the scope of https://dimpact.atlassian.net/browse/PZ-4315. This PR is a first step/task of that story I guess and we should fix that.

Reply:
My question though is: what does the PodiumD 2 release have to do with the version of PostgreSQL that we use in our Docker Compose file? Does it hurt if we are on the latest PostgreSQL version in our Docker Compose file? I would think not. This only becomes tricky if we ever use PostgreSQL specific constructs in ZAC I think. I would personally just use the latest PostgreSQL in our Docker Compose file. We also have our e2e tests to test our environment after all.

Reply:
I agree yes. The problem with weekly cronjobs is that everyone forgets about them. Also should it ever fail you need to wait another week. I would rather stick with nightly jobs for these kind of things because then every morning you know that stuff should have happened..

Reply:
The idea of docker digest pinning is that you don't want to be dependent on what's happening upstream and keep some consistency between runs. 
The images might be updated constantly, sometimes multiple times per day even. That could be triggered by anything, an updated dependent image for example. You don't want to have the digest updated every run or even every day, it's way too noisy (even with automerge).
With the monthly digest refresh at least you'd get the latest security fixes of the base images for image tags that have been not been modified. 
The cron is not external, but it's the internal logic of renovate that's taking care of this.

Reply:
This should end up in the readme